packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -298,7 +300,24 @@ export default class PerformanceMonitoring {
     } else if (event === INVOKE) {
       const data = task.data
       if (data && data.span) {
-        transactionService.endSpan(data.span, data)
+        const { span, response, target } = data
+        let status
+        if (response) {
+          status = response.status
+        } else {
+          status = target.status
+        }
+
+        let outcome = OUTCOME_SUCCESS
+        if (status >= 400 || status == 0) {
```

Comment:
Do we know if the status is 0 for timeouts and aborts as well? I don't remember the values.

Reply:
That's correct, I've added abort as an exception to failure but timeout should be considered failure.

Reply:
I had a conversation with Felix offline that we decided on
Timeout -> `failure`
Abort -> `unknown`